server/src/main/java/com/example/ISAums/dto/request/SendFriendshipRequestRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.example.ISAums.dto.request;
+import com.example.ISAums.model.enumeration.InvitationStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.validation.constraints.NotBlank;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SendFriendshipRequestRequest {
+
+    private UpdateUserProfileRequest sender;
+
+    private UpdateUserProfileRequest invitedUser;
+
+    private InvitationStatus invitationStatus;
+
```

Comment:
sta je pisac htio reci ?

Reply:
ako je ovo slanje friendship request-a, dovoljno je samo staviti `invitedUserId`   i to je to , invitation status je inicijalno FRIENDSHIP_REQUESTED ili kako vec je definisano kada nekog pozoves, a user koji je posalo request je svakao ulogovan tako da to izvlacis iz tokena

Reply:
samo `invitedUserId`